infra-aws/__main__.py

```diff
@@ -23,7 +24,8 @@
 	cidr_block="10.0.101.0/24",
     map_public_ip_on_launch=True,
 	tags={ "Name": "asterion-infra-ec2" },
-	vpc_id=vpc.id
+	vpc_id=vpc.id__(self, name):
+        self.na
```

Comment:
Curious, what is `self.na` being used here?

Reply:
That's not meant to be there! 😅 Looks like I managed to transpose something erroneously here and missed the diff. I've corrected it now.